css/style.css

```diff
@@ -0,0 +1,858 @@
+@import url("https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500;700;900&display=swap");
+
+:root {
+  --white: #ffffff;
+  --red: #ff0000;
+  --card-color: #2f4f4f;
+}
+
+body,
+html {
+  padding: 0;
+  margin: 0;
+  overflow-x: hidden;
+}
+
+* {
+  text-decoration: none;
+  list-style-type: none;
+}
+
+.wrapper {
+  background-color: var(--white);
+}
+
+.site-name {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: bolder;
+}
+
+.topBar {
+  background: #3a3a8d80;
+  display: flex;
+  justify-content: space-around;
+}
+
+.rapidContact {
+  padding: 10px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--white);
+}
+
+.rapidContact span i {
+  color: var(--white);
+  margin-right: 5px;
+}
+
+.searchbar_content {
+  padding: 10px 10px;
+  border-radius: 10px;
+}
+
+.searchbar_btn {
+  margin-left: -3.5px;
+  padding: 10px 10px;
+  border-radius: 25%;
+  background-color: var(--white);
+  border: none;
+}
+
+.searchContent {
+  display: flex;
+  justify-content: flex-end;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 10px;
+  height: 60px;
+}
+
+nav .links {
+  padding: 10px 20px;
+  font-size: 35px;
+  text-transform: capitalize;
+  font-family: "Courier New", Courier, monospace;
+}
+
+.links > a {
+  color: #000000;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  font-weight: 600;
+  font-size: larger;
+}
+
+#active {
+  border-bottom: 3px solid var(--red);
+}
+
+.links a:hover {
+  color: var(--red);
+  border-bottom: 3px solid var(--red);
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.cards {
+  display: flex;
+  gap: 20px;
+}
+
+.card {
+  text-align: center;
+  align-self: normal;
+  border: 3px solid var(--card-color);
+  border-radius: 5px;
+  padding: 15px;
+}
+
+.card .circle img {
+  border-radius: 50%;
+  width: 230px;
+  height: 230px;
```

Comment:
Imaginile tale sunt puțin turtite. Poate le editezi pentru a le face 1:1, sau poți să te joci cu `object-fit`